enctests/test_configs/prores_tests.yml

```diff
@@ -3,7 +3,11 @@ test_prores:
     description: variations of prores_quality
     app: ffmpeg
     suffix: .mov
-    encoding_template: 'ffmpeg {input_args} -i "{source}" -vframes {duration} {encoding_args} -y "{outfile}"'
+    encoding_template: '{ffmpeg_bin} {input_args} -i "{source}" -vframes {duration} {encoding_args} -y "{outfile}"'
+    sources:
+    - sources/enc_sources/chimera_cars_srgb/chimera_cars_srgb.%05d.png.yml
+    - sources/enc_sources/chimera_coaster_srgb/chimera_coaster_srgb.%06d.png.yml
+    - sources/enc_sources/chimera_fountains_srgb/chimera_fountains_srgb.%05d.png.yml
```

Comment:
```suggestion
      - sources/enc_sources/chimera_cars_srgb/chimera_cars_srgb.%05d.png.yml
      - sources/enc_sources/chimera_coaster_srgb/chimera_coaster_srgb.%06d.png.yml
      - sources/enc_sources/chimera_fountains_srgb/chimera_fountains_srgb.%05d.png.yml
```

Reply:
Shouldnt be necessary, it does seem to work without the indent.